.github/workflows/release.yml

```diff
@@ -166,9 +166,9 @@ jobs:
             build-profile: release
             release-file-name-prefix: frequency-rococo-local
           - network: rococo
-            spec: frequency-rococo-testnet
+            spec: frequency-testnet
             build-profile: release
-            release-file-name-prefix: frequency-rococo
+            release-file-name-prefix: frequency-testnet
```

Comment:
frequency-rococo is `frequency-testnet` as it can either start rococo or paseo, so keeping it generic